Packages/eu.netherlands3d.indicators/Runtime/Scripts/UI/Legend.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using Netherlands3D.Indicators.Dossiers;
+using Netherlands3D.Indicators.Dossiers.DataLayers;
+using LegendItemData = Netherlands3D.Indicators.Dossiers.DataLayers.LegendItem;
+using UnityEngine;
+
+namespace Netherlands3D.Indicators.UI
+{
+    public class Legend : MonoBehaviour
+    {
+        [SerializeField] private LegendItem legendItemPrefab;
+
+        [Tooltip("Always clear the old items before showing the new ones")]
+        [SerializeField] private bool clearOld = true;
```

Comment:
Isn't this overkill? Is there a situation where we wouldn't want to clear the old list?